config/initializers/content_security_policy.rb

```diff
@@ -35,7 +29,7 @@
     policy.style_src   :self, *all_domains
 
     policy.frame_src   :self,
-                       *local_domains
+                       *all_domains
```

Comment:
This may be worth removing.

Reply:
Is it worth thinking to be strict like `policy.frame_ancestors :none`?

Reply:
We could, yeah. Especially as we're not using embedded iframes.

Reply:
Also frame_src is deprecated in CSP Level 2 in favor of frame-ancestors. See https://www.w3.org/TR/CSP2/#directive-frame-src